### PR TITLE
Fixes #130

### LINKED ISSUE #130: Broken keywords meta tag
Hi! It seems like in `_config.yml` file you used `keywords` global variable for website/theme keywords meta tag. But in `_includes/head.html` file you used `site.keyword` reference. So, as the result we have empty keywords meta tag on all websites that were generated with this great Jekyll theme. 
Screenshot:
screen-before

I'm going to submit a PR for fixing that in a few minutes ;)

### PR DESCRIPTION
**Pull request by soul-wish**
Screenshot before changes:
screen-before

Screenshot after changes:
screen-after

Thanks for such a great Jekyll theme!

### CODE REVIEW

**Comment by njaremko** [comment]
Thanks for the fix, but FYI google doesn't use meta tags anymore.

**Comment by soul-wish** [comment]
Yeah, I know, thanks. But google is not the only one search engine in our world :), so I hope this broken tag should be fixed anyway.

**Comment by PanosSakkos** [comment]
Great @soul-wish ! 

I'm currently down with the flu. 
As soon as I'm healthy I will merge the PLs and release an new minor version :smile:

**Comment by soul-wish** [comment]
Oh, sure! Get well soon! :wink:

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -7,4 +7,5 @@ In alphabetical order:
   5. Nathan Jaremkio <njaremko@gmail.com>
   6. Panos Sakkos <panos.sakkos@protonmail.com>
   7. Prashant Solanki <prs.solanki@live.com>
-  8. Fatima Villanueva (Kaira) <fzf.villanueva@gmail.com>
\ No newline at end of file
+  8. Sergey Lysenko <soulwish.ls@gmail.com>
+  9. Fatima Villanueva (Kaira) <fzf.villanueva@gmail.com>
```
Comment (PanosSakkos) [inline CONTRIBUTORS.md]: Names should be in alphabetical order, but don't worry I will fix it :wink:

**Pull request merged**